initialize_internal_tide.jl

```diff
@@ -199,60 +201,87 @@ Rig = RichardsonNumber(model; location=(Center, Center, Face), add_background=tr
 ε = KineticEnergyDissipationRate(model)
 χ = TracerVarianceDissipationRate(model, :b)/Bz
 
-slice_diags = (; ε, χ, uhat=û, what=ŵ, v=v, B=B, b=b, Bz=Bz, uhat_z=uz, Rig=Rig)
-point_diags = (; ε, χ, uhat=û, what=ŵ, v=v, B=B, b=b, Bz=Bz, uhat_z=uz, Rig=Rig)
+# set the ouput mode:
+if output_mode == "spinup"
+        checkpoint_interval = 10days
+        slice_diags = (; ε, χ, uhat=û, what=ŵ, v=v, B=B, b=b, Bz=Bz, uhat_z=uz, Rig=Rig)
+        point_diags = (; ε, χ, uhat=û, what=ŵ, v=v, B=B, b=b, Bz=Bz, uhat_z=uz, Rig=Rig)
+        threeD_diags = (; ε, χ, uhat=û, what=ŵ, v=v,  B=B, b=b, Bz=Bz, uhat_z=uz, Rig=Rig)
+
+elseif output_mode == "test"
+        checkpoint_interval = tᶠ
+        slice_diags = (; ε, χ, uhat=û, B=B, b=b, Bz=Bz, uhat_z=uz)
+        threeD_diags = (; ε, χ, uhat=û, what=ŵ,  B=B, b=b, Bz=Bz, uhat_z=uz)
+
+else output_mode == "analysis"
+        checkpoint_interval = 10days
+        threeD_snapshot_interval=4Δtᵒ
```

Comment:
Why hard-code this as a multiple of $\Delta t^{0}$? I feel like this should also be a keyword argument. I also think it should be 1 hour, not 2 hours. 2 hours is not very good resolution for a 12-hour period tidal cycle.